apps/api/src/domain/common/events/coscrad-event-factory.ts

```diff
@@ -5,7 +5,11 @@ import { BaseEvent } from '../../models/shared/events/base-event.entity';
 
 @Injectable()
 export class CoscradEventFactory {
-    constructor(private readonly dynamicDataTypeFinderService: DynamicDataTypeFinderService) {}
+    constructor(private readonly dynamicDataTypeFinderService: DynamicDataTypeFinderService) {
+        if (typeof dynamicDataTypeFinderService.unionFactory?.build !== 'function') {
```

Comment:
We should put a warning here.

Reply:
[Here](https://www.pivotaltracker.com/story/show/188779169) is a story for this.